ui/app/templates/components/license-info.hbs

```diff
@@ -0,0 +1,80 @@
+<PageHeader as |p|>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      {{#link-to "vault.cluster.license" }}
```

Comment:
If this is just a link to this page, I'm not sure this is necessary. Do we make headers links elsewhere?